litellm/model_prices_and_context_window_backup.json

```diff
@@ -675,6 +675,15 @@
         "litellm_provider": "mistral",
         "mode": "embedding"
     },
+    "mistral/mistral-7b-instruct-v0.2": {
```

Comment:
Hey @Victorivus i don't see this here https://docs.mistral.ai/platform/pricing/#operation/listModels

Can you share docs on this?

Reply:
Sorry I arrived here from a project that uses this list to get information on max_tokens etc. and didn't realize it is a commercially available list of endpoints.

https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2

Since Mistral does not offer it on their platform I guess you do not add it to the list ?

Reply:
@Victorivus oh - can you share the project? 

In our helper function - `get_max_tokens()` https://docs.litellm.ai/docs/completion/token_usage#6-get_max_tokens

we do check the huggingface hub config and get the hf models' max tokens from there